File: app/src/main/java/com/hqq/example/ui/jetpack/databinding/DataBanding.java
package com.hqq.example.ui.jetpack.databinding;

import android.view.View;

import com.hqq.core.utils.ToastUtils;
import com.hqq.core.utils.log.LogUtils;

/**
 * @Author : huangqiqiang
 * @Package : com.hqq.example.ui.jetpack.viewbinding
 * @FileName :   ViewBanding
 * @Date : 2020/7/21 0021  下午 3:03
 * @Email : qiqiang213@gmail.com
 * @Descrive :
 */
public class DataBanding {
    public void onClickButton(View v) {
        ToastUtils.showToast("onClickButton");
        LogUtils.e("-----------onClickButton");

    }

    public void onClickButton4Text(String name) {
        ToastUtils.showToast("onClickButton   " +name);
        LogUtils.e("-----------onClickButton4Text");
    }
}

File: app/src/main/java/com/hqq/example/ui/jetpack/databinding/DataBanding.kt
package com.hqq.example.ui.jetpack.databinding

import android.view.View
import com.hqq.core.utils.ToastUtils.showToast
import com.hqq.core.utils.log.LogUtils.e

/**
 * @Author : huangqiqiang
 * @Package : com.hqq.example.ui.jetpack.viewbinding
 * @FileName :   ViewBanding
 * @Date : 2020/7/21 0021  下午 3:03
 * @Email : qiqiang213@gmail.com
 * @Descrive :
 */
class DataBanding {
    fun onClickButton(v: View?) {
        showToast("onClickButton")
        e("-----------onClickButton")
    }

    fun onClickButton4Text(name: String) {
        showToast("onClickButton   $name")
        e("-----------onClickButton4Text")
    }
}